.circleci/config.yml

```diff
@@ -18,15 +18,24 @@ aliases:
         if [ -n "$CIRCLE_PR_NUMBER" ]; then
           .ci/setup.sh
         fi
+  - &cache_calc_key
+    run:
+      name: Calculate cache key
+      command: |
+        if [ -n "$CIRCLE_PR_NUMBER" ]; then
+          date +%Y-%m > .circleci_cachekey
+        else
+          echo not running > .circleci_cachekey
```

Comment:
It should run for nix-build / stack-build. Does this logic break caching for those?

Reply:
Yes, you're right